uk.co.piggz.amazfish.appdata.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <name>Amazfish</name>
+  <summary>Smart watch sync app</summary>
+  <id>uk.co.piggz.amazfish</id>
+  <launchable type="desktop-id">harbour-amazfish-ui.desktop</launchable>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0</project_license>
+  <description>
+    <p>
+        Companion application for Huami Devices (such as Amazfit Bip, Cor, MiBand2/3 and GTS and GTS) and the Pinetime Infinitime.
+    </p>
+  </description>
+  <url type="homepage">https://github.com/piggz/harbour-amazfish</url>
+  <url type="bugtracker">https://github.com/piggz/harbour-amazfish/issues</url>
+  <url type="donation">https://paypal.me/piggz</url>
+
+  <requires>
+    <display_length compare="ge">small</display_length>
+  </requires>
+<recommends>
+  <control>keyboard</control>
+  <control>pointing</control>
+  <control>touch</control>
+</recommends>
+
+  <developer id="@piggz@fosstodon.org">
+      <name>Adam Pigg</name>
+  </developer>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://github.com/piggz/harbour-amazfish/raw/master/screenshots/screenshot1.png</image>
```

Comment:
These should be from a commit or a tag

Reply:
simmilar to my comment about .desktop file. I guess I can open pull request, but it will take some time to have tagged version which can be published. My experience is that Adam is not much excited into other distributions and he forgets to update metadata for other packaging then he is interested. Maintaining appdata in flathub repository will probably improve quality of metadata.

Reply:
replaced to tag. Thanks for feedback